src/app/space/create/deployments/apps/deployment-details.component.spec.ts

```diff
@@ -67,45 +72,33 @@ describe('DeploymentDetailsComponent', () => {
     mockSvc.deleteApplication.and.returnValue(Observable.of('mockDeletedMessage'));
 
     mockEnvironment = { name: 'mockEnvironment' } as Environment;
+  });
 
-    TestBed.configureTestingModule({
-      imports: [CollapseModule.forRoot()],
-      declarations: [
-        DeploymentDetailsComponent,
-        FakeDeploymentsDonutComponent,
-        FakeDeploymentGraphLabelComponent,
-        FakePfngChartSparkline
-      ],
-      providers: [
-        { provide: DeploymentsService, useValue: mockSvc }
-      ]
-    });
-
-    fixture = TestBed.createComponent(DeploymentDetailsComponent);
-    component = fixture.componentInstance;
-
+  initContext(DeploymentDetailsComponent, HostComponent, {
+    imports: [CollapseModule.forRoot()],
+    declarations: [
+      DeploymentDetailsComponent,
```

Comment:
I think you can skip declaring the tested component itself here - the initContext util will add it to the declarations for you automatically.